packages/io-services-cms-models/src/lib/fsm/store.cosmos.ts

```diff
@@ -82,7 +82,7 @@ export const createCosmosStore = <
       TE.map((operationResponses) =>
         operationResponses.map(
           flow(
-            O.fromPredicate((res) => res.statusCode === 404),
+            O.fromPredicate((res) => res.statusCode === 200),
```

Comment:
maybe it's worth a unit test?

Reply:
a ticket to test store.cosmos.ts has been created